be/src/runtime/exec_env.cpp

```diff
@@ -212,6 +212,15 @@ Status ExecEnv::_init(const std::vector<StorePath>& store_paths, bool as_cn) {
     }
     _query_rpc_pool = new PriorityThreadPool("query_rpc", query_rpc_threads, std::numeric_limits<uint32_t>::max());
 
+    // The _load_rpc_pool now handles routine load RPC and table function RPC.
+    RETURN_IF_ERROR(ThreadPoolBuilder("load_rpc") // thread pool for load rpc
+                            .set_min_threads(10)
+                            .set_max_threads(1000)
```

Comment:
1000 maybe too high ?

Reply:
The user may have hundreds of routine load jobs, which need hundreds of threads. This only set the upper limit of the thread pool. In most cases, the number of threads would be much less.